go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -25,6 +25,11 @@ const (
 	egressFirewallUpdateError      = "EgressFirewall Rules not correctly updated"
 )
 
+var (
+	unknownIPv4Destination = "0.0.0.0/0"
```

Comment:
I don't know, I googled it and according to wikipedia [1]

>  0.0.0.0 is a non-routable meta-address used to designate an invalid, unknown or non-applicable target.

[1]: https://en.wikipedia.org/wiki/0.0.0.0

Reply:
0.0.0.0/0 as all IP addresses is a holdover from openshift/sdn

Reply:
in my experience it means all IPs:
In the context of routing tables, a network destination of 0.0.0.0 is used with a network mask of 0 to depict the default route as a destination subnet. This destination is expressed as "0.0.0.0/0" in CIDR notation. It matches all addresses in the IPv4 address space and is present on most hosts, directed towards a local router.

Since we are using it for that purpose can we change the name?

Reply:
OK, I will change the name